### PR TITLE
 	rerun monitoring_location to get renamed column, add fk

### PR DESCRIPTION
**Pull request by kmschoep-usgs**
make create_date and create_user not nullable.

### CODE REVIEW

File: mlr-database-schema/src/main/liquibase/test_data.xml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/test_data.xml]: woops, prob not supposed to check this in

File: mlr-database-schema/src/main/liquibase/tables/monitoring_location/changeLog.yml [hunk 2]
```diff
@@ -5,7 +5,7 @@ databaseChangeLog:
 
   - changeSet:
       author: kmschoep
-      id: "create.table.monitoring_location"
+      id: "create.table.monitoring_location2"
```
Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/tables/monitoring_location/changeLog.yml]: prob don't want that 2 in there either

File: mlr-database-schema/src/main/liquibase/test_data.xml (hunk 1 above)
Comment (kmschoep-usgs) [inline mlr-database-schema/src/main/liquibase/test_data.xml]: woops, probably not!  How to get it out?

Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/test_data.xml]: yeah

File: mlr-database-schema/src/main/liquibase/tables/monitoring_location/changeLog.yml (hunk 2 above)
Comment (kmschoep-usgs) [inline mlr-database-schema/src/main/liquibase/tables/monitoring_location/changeLog.yml]: Need those in there so can rerun the script after dropping the monitoring_location table in local postgres.  I can't get the permissions of databasechangelog to let me delete rows.

Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/tables/monitoring_location/changeLog.yml]: ahh ok! i just blew it all away locally but i understand where you're coming from!

File: mlr-database-schema/src/main/liquibase/tableDDL.xml [hunk 3]
```diff
@@ -1,58 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/tableDDL.xml]: does this mean the whole file was deleted?

Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/tableDDL.xml]: your commit says yes, just making sure!

Comment (kmschoep-usgs) [inline mlr-database-schema/src/main/liquibase/tableDDL.xml]: Yep, it was my first attempt at all this.

**Comment by mhines-usgs** [comment]
:+1:

**Pull request merged**